src/SAWScript/Interpreter.hs

```diff
@@ -3780,6 +3785,12 @@ primitives = Map.fromList
     , " 1 = basic debug output, 2 = verbose debug output,"
     , " 3 = all debug output" ]
 
+  , prim "prove_refinement" "ProofScript ()"
```

Comment:
Naming quibble: most (and quite possibly all) of the existing `prove_*` functions in SAWScript inhabit `TopLevel`, as they meant to be the things that actually perform proofs. `prove_refinement` is an exception to this convention, as it is a proof _script_, not a thing that performs the proof on its own.

How would you feel about simply calling this "`refinement`"?

Reply:
Ah, that's an excellent point!

I looked at a few other `ProofScript` commands in `Interpreter.hs`, and it looks like the pattern is that the Haskell functions `proveZ3`, `proveRME`, etc. map to the SAW commands `z3`, `rme`, etc. – so following that, indeed our `proveRefinement` should indeed be `refinement` in SAW.

However, a command named `refinement` really doesn't strike me as the right thing here. Maybe `mrsolver`? `solve_refinement`? @eddywestbrook Do you have thoughts? (Regardless, I agree we shouldn't start our command with `prove_`.)

Reply:
We also have a variety of other `ProofScript` commands that begin with `goal_*` (e.g., `goal_eval`), so `goal_refinement` is another option.

Reply:
After some discussion, Matt and I decided on `mrsolver` as the command name. That seems to fit with how tactics for calling other solvers and provers are named, as the name of the solver itself.